commands/command_lock.go

```diff
@@ -30,8 +30,14 @@ func lockCommand(cmd *cobra.Command, args []string) {
 
 	success := true
 	locks := make([]locking.Lock, 0, len(args))
+	lockData, err := computeLockData()
+	if err != nil {
+		lockClient.Close()
+		Error(err.Error())
+		os.Exit(3)
```

Comment:
I always like to use new, different error codes so that we can distinguish various cases, which I think is valuable to the user.  I hadn't intended to add any new tests for this case, though.  If you prefer, I can change it back.